src/Microsoft.AspNetCore.Sockets.Client.Http/HttpConnection.cs

```diff
@@ -116,6 +121,10 @@ private Task StartAsyncCore()
                     if (t.IsFaulted)
                     {
                         _startTcs.SetException(t.Exception.InnerException);
+                        if (_closedTcs.Task.IsFaulted)
```

Comment:
Why is there new code here that wasn't here before?

Reply:
@mikaelm12 can you add a comment there. I'm also not sure why we're checking the state of the close task.